lib/src/internal_style.dart

```diff
@@ -66,4 +66,13 @@ abstract class InternalStyle extends Style {
 
   /// Returns the URI that represents [path], which is assumed to be absolute.
   Uri absolutePathToUri(String path);
+
+  /// Returns whether [codeUnit1] and [codeUnit2] are considered equivalent for
+  /// this style.
+  bool codeUnitsEqual(int codeUnit1, int codeUnit2) => codeUnit1 == codeUnit2;
+
+  /// Returns whether [path1] and [path2] are textually equivalent.
```

Comment:
"are textually equivalent" sounds like the opposite of what this does to me. It's for handling cases where the paths are textually different but not different according to the file system. How about "can be distinguished by the file system"?